README.md

```diff
@@ -38,91 +32,300 @@ IncludeOS is free software, with "no warranties or restrictions of any kind".
 
-### Testing the installation
+To ensure the profile/s has been installed do:
 
-A successful setup enables you to build and run a virtual machine. There are a few demonstration services in the source folder. If you look in the `examples/` folder you see these. If you enter `demo_service` and type `boot --create-bridge .` this script will build the service and boot it using [qemu].
+```
+    $ conan profile list
+```
 
+Verify the content of your profile by:
 ```
-    $ cd examples/demo_service
-    $ boot --create-bridge .
+    $ conan profile show <yourprofilename>
 ```
 
-will build and run [this example service](./examples/demo_service/service.cpp). You can visit the service on [http://10.0.0.42/](http://10.0.0.42/).
+If your profile is on the list and contents are verified, you are set to use the
+profile for building.
 
-More information is available on our [documentation site](http://includeos.readthedocs.io/en/latest/Getting-started.html#testing-the-example-service).
+###### IncludeOS Bintray Repositories
 
-### Writing your first service
+The [Bintray](https://bintray.com/includeos) repository is where all the packages used to build IncludeOS
+are uploaded. Adding the repo to your conan remotes will give you access to all
+our packages developed for IncludeOS.
 
-1. Copy the [./seed/service](./seed/service) directory to a convenient location like `~/your_service`. Then, just start implementing the `Service::start` function in the `Service` class, located in [your_service/service.cpp](./seed/service/service.cpp) (very simple example provided). This function will be called once the OS is up and running.
-2. Update the [CMakeLists.txt](./seed/service/CMakeLists.txt) to specify the name of your project, enable any needed drivers or plugins, etc.
+You check your repository remotes, do:
 
-**Example:**
+```
+conan remote list
+```
+If the includeOS-Develop remote is not added do, you have to add it.
+
+To add the IncludeOS-Develop conan Bintray repository to your conan remotes:
 
 ```
-    $ cp -r seed/service ~/my_service
-    $ cd ~/my_service
-    $ emacs service.cpp
-    ... add your code
-    $ mkdir build && cd build
-    $ cmake ..
-    $ make
-    $ boot my_service
+conan remote add includeos-test https://api.bintray.com/conan/includeos/test-packages
 ```
 
-Take a look at the [examples](./examples) and the [tests](./test). These all started out as copies of the same seed.
+##### Build IncludeOS
+
+Finally to build IncludeOS do:
+
+```
+    $ conan create <path-to-conan-recipe> <conan-user>/<conan_channel> -pr <profile-name>
+```
+
+To build **IncludeOS on Linux** use profile named `clang-6.0-linux-x86_64` :
+
+```
+    $ conan create IncludeOS includeos/test -pr clang-6.0-linux-x86_64
+```
+
+To build **IncludeOS on Linux** for **Macos** use profile named `clang-6.0-macos-x86_64`.
+
+_To build **IncludeOS on Mac** requires a number of other configurations, which
```

Comment:
@staiyeba / @KristianJerpetjon we're not releasing with this of course? If you want this merged now and change it again before release I'm fine with that as long as we remember.

Reply:
I can remove this line for the release if that is preferred. 
In that case, I will leave a `TODO:`  So we remember to update it.